tests/unit/test_space.py

```diff
@@ -21,6 +22,28 @@
 from trieste.space import SearchSpace, DiscreteSearchSpace, Box
 
 
+class Line(SearchSpace):
+    def __init__(self, limit: float):
+        self._limit = tf.constant(limit)
```

Comment:
someone might down the line want to do `line.limit * [1, 2, 3]`, which would, much to their confusion, give `[[1, 2, 3], [1, 2, 3], [1, 2, 3]]` rather than `tf.Tensor([ 5, 10, 15])`